pkg/certmanager/certmanager.go

```diff
@@ -75,12 +78,45 @@ type Signer interface {
 	Sign(ctx context.Context, identities string, csrPEM []byte, duration time.Duration, usages []cmapi.KeyUsage) (Bundle, error)
 }
 
+// IssuerChangeSubscription is a subscription that can be used to get changes
+// to issuer config
+type IssuerChangeSubscription struct {
+	C <-chan *cmmeta.ObjectReference
+
+	// The same channel as above is mirrored in c, but without the "<-"
+	// restriction. This allows the channel to be written to by this package.
+	c chan *cmmeta.ObjectReference
+
+	lock   sync.Mutex
+	closed bool
+}
```

Comment:
suggestion: I'm a bit squeamish about having two variables with the same name differing only in casing - I personally find that pretty hard to keep in my head.

I like `C` for the exported one, but maybe something like `sendCh`, `internalCh` or something else instead of `c`?